js/sign/README.md

```diff
@@ -1,18 +1,85 @@
 # Signing with Integrity Block
+
 This is a Node.js module for signing [Web
 Bundles](https://wpack-wg.github.io/bundled-responses/draft-ietf-wpack-bundled-responses.html)
 with [integrityblock](../../explainers/integrity-signature.md).
 
 The module takes an existing bundle file and an ed25519 private key, and emits a
 new bundle file with cryptographic signature added to the integrity block.
 
+## Installation
+
+Using npm:
+
+```bash
+npm install wbn-sign
+```
+
+## Usage
+
+Please be aware that the APIs are not stable yet and are subject to change at
+any time.
+
+Signing a web bundle file:
+
+```javascript
+import * as fs from 'fs';
+import * as wbnSign from 'wbnSign';
+
+// Read an existing web bundle file or generate using "wbn" npm package.
+const webBundle = fs.readFileSync('./path/to/webbundle.wbn');
+
+const privateKey = wbnSign.parsePemKey(
+  fs.readFileSync('./path/to/privatekey.pem', 'utf-8')
+);
+
+const integrityBlock = new wbnSign.IntegrityBlockSigner(webBundle, {
+  key: privateKey,
+}).sign();
+
+var fd = fs.openSync('signed-webbundle.swbn', 'w');
+fs.writeSync(fd, integrityBlock);
+fs.writeSync(fd, webBundle);
+fs.closeSync(fd);
```

Comment:
This unnecessarily exposes the fact that a signed bundle is the concatenation of an integrity block and an unsigned bundle. Also, this wouldn't work when `webBundle` already had an integrity block (`IntegrityBlockSigner` currently does not support that case though).

It would be nice if `IntegrityBlockSigner` provided a function that returns the entire signed bundle as a single `Uint8Array` or `Buffer`.